bot/cogs/sudo.py

```diff
@@ -44,72 +52,64 @@ def get_uptime(self) -> str:
         else:
             return f"{hours} hr, {minutes} mins, and {seconds} secs"
 
-    async def is_owner(ctx: Context) -> t.Union[bool, None]:
-        if ctx.author.id in config.devs:
-            return True
-        else:
-            embed = Embed(description="This is an owner-only command, you cannot invoke this.", color=Color.red())
-            await ctx.send(embed=embed)
-
     @group(hidden=True)
     async def sudo(self, ctx: Context) -> None:
         """Administrative information."""
         pass
 
     @sudo.command()
-    @check(is_owner)
+    @check(is_sudo)
```

Comment:
This name doesn't make any sense
Sudo stands for **Su**peruser**Do**
And this command now stands for `is_superuserdo`